prow/jobs/incubator/compass/compass-integration.yaml

```diff
@@ -39,27 +37,15 @@ presubmits: # runs on PRs
     - ^master$
     <<: *vm_job_template
     # following regexp won't start build if only Markdown files were changed
-    # run_if_changed: "^((chart/\\S+|installation/\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))" # TODO: Uncomment when done testing
+    run_if_changed: "^(chart/\\S+|installation/\\S+)\\.([^.\\s]|[^.\\s]{3,}|[^.\\s][^dD]|[^mM][^.\\s])$"
```

Comment:
please simplify this regexp - IMO it is better to have simple regexp even if sometimes we run integration tests if we modified only documentation files